.env.example

```diff
@@ -1,5 +1,6 @@
 export WEB3_ALCHEMY_PROJECT_ID="0-USE-YOUR-OWN-ALCHEMY-PROJECT-ID-HERE"
 export ETH_PRIVATE_KEY_BE_CAREFUL="0x123-USE-YOUR-OWN-PRIVATE-KEY-HERE"
+export WEB3_FANTOM
```

Comment:
@Lesigh-3100 ? Given that it's a free RPC, is there a key required? If so, why is `WEB3_FANTOM` formatted differently here than the others?

Reply:
I formatted it differently because it's not using Alchemy. 

A key isn't required, but should be an option. For public usage, the value in .env should be "public".

Reply:
This seems to be the only place where the `WEB3_FANTOM` environment variable is used:
```
WEB3_ALCHEMY_PROJECT_ID = os.environ.get("WEB3_FANTOM")
```
Have you verified that it achieves the desired purpose when `WEB3_FANTOM` is set to `"public"`?

Reply:
"public" serves as the API key field - it is the correct input for the public API endpoint. Inputting it this way allows the user to change "public" with a paid API key if they desire. 

It achieves this by combining this key to the RPC_ENDPOINT in provider.py.

Reply:
So why not use the same (or a similar) example appearing two lines above that one:
```
export WEB3_ALCHEMY_PROJECT_ID="0-USE-YOUR-OWN-ALCHEMY-PROJECT-ID-HERE"
```
?

Reply:
We need a distinct API key name for Fantom.

Reply:
I've updated the documentation for this - please let me know if you think it's clear enough now!